lnwallet/channel_test.go

```diff
@@ -7638,3 +7638,45 @@ func TestChannelMaxFeeRate(t *testing.T) {
 	assertMaxFeeRate(0.000001, 690)
 	assertMaxFeeRate(0.0000001, chainfee.FeePerKwFloor)
 }
+
+// TestChannelFeeRateFloor asserts that proposing commitments using
+// chainfee.FeePerKwFloor results in a exchange of commitments.
+func TestChannelFeeRateFloor(t *testing.T) {
+	t.Parallel()
+
+	aliceChannel, bobChannel, cleanUp, err := CreateTestChannels(
+		channeldb.SingleFunderTweaklessBit,
+	)
+	if err != nil {
+		t.Fatalf("unable to create test channels: %v", err)
+	}
+	defer cleanUp()
+
+	// Set the fee rate to the proposing fee rate floor.
+	minFee := chainfee.FeePerKwFloor
+
+	setMinFee := func(alice, bob *LightningChannel) {
+		if err := alice.UpdateFee(minFee); err != nil {
+			t.Fatalf("unable to send fee update")
+		}
+		if err := bob.ReceiveUpdateFee(minFee); err != nil {
+			t.Fatalf("unable to receive fee update")
+		}
+
+		// Check that we can still sign commitments.
+		sig, htlcSigs, _, err := alice.SignNextCommitment()
+		if err != nil {
+			t.Fatalf("alice unable to sign commitment: %v", err)
+		}
+
+		// Check that we can still receive commitments.
+		err = bob.ReceiveNewCommitment(sig, htlcSigs)
+		if err != nil {
+			t.Fatalf("bob unable to process alice's new "+
+				"commitment: %v", err)
+		}
+	}
+
+	setMinFee(aliceChannel, bobChannel)
+	setMinFee(bobChannel, aliceChannel)
```

Comment:
Only one of Alice, Bob is the initiator so calling `UpdateFee` should fail

Reply:
you're right, fixed!